.browserslistrc

```diff
@@ -1,2 +1 @@
 last 2 versions
```

Comment:
Might want to change this while we are here to `defaults`

https://github.com/browserslist/browserslist#best-practices

Reply:
Sounds good to me 👍🏼